scrapy/contrib/loader/processor.py

```diff
@@ -63,6 +63,33 @@ def __call__(self, values):
         return values
 
 
+class JmesProcessor(object):
+    """
+        Query the input string for the jmespath (given at instantiation), and return the answer
+        Requires : jmespath(https://github.com/boto/jmespath), ast
+        Note: JmesProcessor accepts only a single string at a time and returns string/dict/None based on the jmespath query.
+    """
```

Comment:
* Could you please fix the indentation? 
* According to PEP8 docstrings should be wrapped at 72th column.
* It would be nice for it to follow http://sphinx-doc.org/domains.html#info-field-lists